parmed/rdkit/rdkit.py

```diff
@@ -36,6 +36,8 @@ def from_smiles(smiles, coordinates=True):
         smiles : str, smiles
         coordinates : bool, default True
             if True, use `rdkit.Chem.AllChem.EmbedMultipleConfs to assign coordinates
+        hydrogens : bool, default False
+            if True, use `rdkit.Chem.AddHs` to generate explicit hydrogens
```

Comment:
IMO `hydrogens=True` is the better default (I know it's not strictly backwards-compatible, but the next release will be a major 4.0 release anyway, so let's get all of the breaking changes in now that we can).

```suggestion
        hydrogens : bool, default True
            if True, use `rdkit.Chem.AddHs` to generate explicit hydrogens
```

Reply:
I agree. Personally, when I’m using SMILES strings, they are pretty much always some carbon based structures, and adding explicit hydrogens just makes life so much easier.